src/blech32.h

```diff
@@ -19,16 +20,33 @@
 namespace blech32
 {
 
+enum class Encoding {
+    INVALID,  //!< Failed decoding
+
+    BLECH32,  //!< Original blech32
+    BLECH32M, //!< Blech32m tweaked a la bech32
+};
+
 /** Encode a Bech32 string. Returns the empty string in case of failure. */
```

Comment:
Nit: Bech->Blech.

(Upstream has a longer comment here, but this was already true before this change -- I don't know if it's untrue for us for some reason. Converted for our use it would be:

`/** Encode a Blech32 or Blech32m string. If hrp contains uppercase characters, this will cause an
 *  assertion error. Encoding must be one of BLECH32 or BLECH32M. */`)